dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -695,7 +725,7 @@ public Token addBitbucketToken(@ApiParam(hidden = true) @Parameter(hidden = true
             return tokenDAO.findById(create);
         } else {
             LOG.info("Could not find user");
-            throw new CustomWebApplicationException("User not found", HttpStatus.SC_CONFLICT);
+            throw new CustomWebApplicationException("User not found", HttpStatus.SC_NOT_FOUND);
```

Comment:
hmmm, not found does seem to make more sense than conflict, without this is a conflict possible anymore? 
otherwise, `@ApiResponse(responseCode = HttpStatus.SC_CONFLICT + "", description = "Conflict")` seems superfluous

Reply:
Yeah it's still possible, there's a conflict a couple lines before this